server.js

```diff
@@ -18,6 +18,7 @@ app.use('/register', require('./routes/register'));
 app.use('/login', require('./routes/login'));
 app.use('/logout', require('./routes/logout'));
 app.use('/mail-verification', require('./routes/verifyEmail'));
+app.use('/profile', require('./routes/profile'));
```

Comment:
Add the `require` imports at the top of the file, and put those routes in the `app.use`.
Like this:

```

const loginRoutes = require('./routes/login');
app.use('/login', loginRoutes);

```

Reply:
make another PR with those changes